bin/upload_new_release_to_s3.py

```diff
@@ -0,0 +1,48 @@
+import boto3
+import os
+import datetime
+import argparse
+
+parser = argparse.ArgumentParser(description='Usage')
+parser.add_argument('-r','--release-name', type=str,
+                    help="Name of release")
+args = parser.parse_args()
+
+if not args.release_name:
+  raise Exception("Please include a name for your release")
+
+s3 = boto3.client('s3')
+now = datetime.date.today()
```

Comment:
to avoid the issue you ran into below, I would recommend introducing a new variable here 
```
release_name = f"{now}-{args.release_name}"
```